libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java

```diff
@@ -523,17 +523,20 @@ private static boolean resizeImageAndWriteToStream(Context context,
 
         boolean result = bmpRotated.compress(fmt, quality, outStream);
         bmpRotated.recycle();
+        bmpResized.recycle();
```

Comment:
I think we don't even need to call recycle() as we have a minSdk of 16.
Check this https://developer.android.com/topic/performance/graphics/manage-memory.html

`Android 2.3 adds concurrent garbage collection, which means that the memory is reclaimed soon after a bitmap is no longer referenced.`